### ISSUE TITLE
DRAG FROM ONE GRID TO ANOTHER

### ISSUE THREAD

**Issue by bpbhat77** [ducksboard/gridster.js#259]
Can we DRAG FROM ONE GRID TO ANOTHER in gridster 

i.e we have 2 gridster instances gridster1 and gridster2 and on drag and drop from gridster1 to gridster2 it should move grid element from gridster1 to gridster2

**Comment by Janet-ten-Boom** [comment]
http://www.jqueryscript.net/demo/Dynamic-Drag-Drop-Grid-Layout-Plugin-shapeshift/demo/

**Comment by davidpelayo** [comment]
I don't think that just by posting a link to a shapeshift JS solution is a correct answer to this issue. Additionally, shapeshift solution arranges automatically the elements so that means it's an entirely different solution.

To achieve this, I consider it should be combined a set of draggable, droppable, removeWidget and addWidget in, being able to move a widget from one grid to another. 

Do you have any idea if this is something planned to be integrated soon?

**Comment by Janet-ten-Boom** [comment]
Haha, no it wasn't supposed to be a full and complete answer.
But your initial question was if the dragging and dropping from one grid to another was possible, or at least I read it that way. 
I came across shapeshift and I thought that maybe by looking at other products and their code this might get you thinking towards a solution in a direction you wouldn't have thought of otherwise. 
(Pff...that's really a bad sentence). 
I'm really a foetus (not even a baby) in programming world,  so this is it. I'm crossing the ocean in a bathtub, so to speak. I hope someone else can help you out. 

Greetsz!
Janet

**Comment by bpbhat77** [comment]
Thanks for link 

In shapeshift I want to add  grid inside grid ,like in demo we have 2 grids , instead of that i want many grids which can be movable(drag/drop).

I tried  which shapeshift by adding  div class="container" inside  div class="container" but grid inside grid is not working :(

anyways thanks

**Comment by rochapablo** [comment]
Does anyone have done this work?
